grpc/server/cmake/helpers/build_target.cmake

```diff
@@ -5,8 +5,17 @@ function(build_target SRC_DIR TARGET_DIR)
         COMMAND ${CMAKE_COMMAND}
             -G${CMAKE_GENERATOR}
             -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+            "-DCMAKE_FIND_ROOT_PATH=${CMAKE_FIND_ROOT_PATH}"
```

Comment:
The `"` are a bit ugly but I guess that's required?

Reply:
Yes, unfortunately it is, for lists :-). I just tried to rebuild without the quotes and it fails.